js/food.js

```diff
@@ -0,0 +1,62 @@
+// XMLHttpRequests are REALLY HARD and I hope I did this in a sustainable manner
+
+// Elements for inserting information into the Dog Food Table in the DOM
+var products = document.getElementById('products');
+var dogBrands = document.getElementsByClassName('brand-dog');
+var dogType = document.getElementsByClassName('types-dog');
+
+// Nesting ALL the for-loops to get the objects to appear in the order I want in the DOM
+// STILL SURPRISED THAT THIS WORKS, WHAT KIND OF WITCHCRAFT IS THIS?!
+function putTheDogStuffintheDom () {
+  var dogObjects = JSON.parse(dogs.responseText);
+  console.log(dogObjects);
+  for (var i = 0; i < dogObjects.dog_brands.length; i++) {
+    dogBrands.item(i).innerHTML += dogObjects.dog_brands[i].name;
```

Comment:
Small suggestion to make your code cleaner (and less painful to work with):
Once you're in the ```for``` loop, you can save off the element you're at in the loop as a variable, like ```var brand = dogObjects.dog_brands[i]``` and then refer to _that_ further down the chain, and do the same at each level.

For instance, here, if you'd made that variable, ```dogBrands.item(i).innerHTML += dogObjects.dog_brands[i].name;``` becomes ```dogBrands.item(i).innerHTML += brand.name;``` and further on...